
message RPCRequest_pb2 {
    required int32 callid = 1;
    required string methodName = 2;
    required bytes params = 3;
}


message RPCResponse_pb2 {
    required int32 callid = 1;
    required bytes retvalue = 2;
}
